packages/gatsby-theme-api-docs/src/hooks/use-api-resources.js

```diff
@@ -3,7 +3,40 @@ import { useStaticQuery, graphql } from 'gatsby';
 export default () => {
   const queryResult = useStaticQuery(
     graphql`
-      {
+      fragment portOrPutMethod on RamlResourcePost {
```

Comment:
Thanks. Can we also try to include the other methods get,delete? Most of the fields are the same. We would probably need to split it up in multiple smaller fragments.

Reply:
```suggestion
      fragment postOrPutMethod on RamlResourcePost {
```

I guess you mean "post", not "port"? 

although "get", "options" and "head" do not support a request body in HTTP  they are modelled uniformly in RAML and can (+ should) have the identical data model in our implementaiton.  What you have here is rather a "httpMethod" fragment and not a method-specific one.

Reply:
I think it's best we do that in another PR, only way I see a proper implementation is

- redefine the method schemas to extend a common interface
- write fragments and queries

Reply:
Sure, this can be a follow up. Thanks